Cli/Application.php

```diff
@@ -27,8 +27,8 @@
 
 class Application extends SymfonyApplication
 {
-    const APPLICATION_NAME = 'SymfonyInsight CLI';
-    const APPLICATION_VERSION = '1.7.4';
+    public const APPLICATION_NAME = 'SymfonyInsight CLI';
+    public const APPLICATION_VERSION = '1.7.4';
```

Comment:
typed props are 7.4, but we want to be able to run on 7.2
did php-cs-fixer suggest the change? then we should disable the corresponding rule